src/main/java/com/google/codeu/servlets/ChartServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.JsonArray;
+import com.google.codeu.utils.CsvLoader;
+
+@WebServlet("/moviechart")
+public class ChartServlet extends HttpServlet {
+
+  private JsonArray movieInfoArray;
```

Comment:
nit: Not necessary to add the type to the name as the type is already specified when defining this variable. `movieInfos` or something would be better.

Reply:
Names refactored